ios/Showcase/Targets/AppUIKit/Sources/verifier/VerifyCwt.swift

```diff
@@ -0,0 +1,58 @@
+import SwiftUI
+import SpruceIDMobileSdkRs
+import SpruceIDMobileSdk
+
+struct VerifyCwt: Hashable {}
+
+struct VerifyCwtView: View {
+    
+    @State var success: Bool?
+    @State var credentialPack: CredentialPack?
+    @State var code: String?
+    @State var error: Error?
+    
+    @Binding var path: NavigationPath
+    
+    var body: some View {
+        if success == nil {
+            ScanningComponent(
+                path: $path,
+                scanningParams: Scanning(
+                    subtitle: "Scan the QR Code",
+                    scanningType: .qrcode,
+                    onCancel: {
+                        path.removeLast()
+                    },
+                    onRead: { code in
+                        Task {
+                            do {
+                                credentialPack = CredentialPack()
+                                let cwt = try Cwt.newFromBase10(payload: code)
+                                _ = credentialPack!.addCwt(cwt: cwt)
```

Comment:
Why do we add it to a credentialPack? We're just verifying, not storing

Reply:
It's for display only, we are indeed not storing.